### ISSUE TITLE
[Suggestion] Option that allow's sticky windows to penetrate fullscreen windows

### ISSUE THREAD

**Issue by onlyreportingissues** [pop-os/cosmic-comp#534]
E.g. having a YouTube video or a picture in fullscreen mode will ignore any sticky window (which can be fine for some sticky windows).

Having the option to force some sticky windows to penetrate also fullscreen windows would be nice though (would be useful e.g. when one has a video chat window opened, but still wants to do something else in a fullscreen window at the same time).

**Comment by onlyreportingissues** [comment]
It feels more pleasant than maximized windows, since fullscreen windows cut out also the dock & panel and also use the entire screen for the video (while a maximized window on YouTube e.g. won't maximize the video, but only the YT page the video is shown at while the video keeps it non-fullscreen mode).

The main purpose for me is being able to use the entire screen for a video while having the option to show a little video chat window at some part of the screen also.

**Comment by D-Brox** [comment]
I agree, using small sticky windows over a video fullscreen is something I used a lot on gnome when doing watchalongs, so there is a use-case for this kind of behavior

**Comment by bobcat141** [comment]
Another use case is for being able to watch a video while playing a fullscreen game (or being able to keep a text editor visible at all times for note-taking).
